client/src/components/Register/SelectMethod.js

```diff
@@ -56,17 +56,17 @@ class SelectMethod extends Component {
     const { highlightedMethod } = this.state;
 
     return (
-      <div className="mfa-method-tile__actions">
+      <div className="mfa-actions">
         <button
-          className="mfa-method-tile__action-next btn btn-success"
+          className="mfa-actions__action mfa-method-tile__action--next btn btn-success"
```

Comment:
Shouldn't this be `mfa-method-tile-action__next`?

Reply:
I’ve converted next and back to modifiers rather than elements